### ISSUE TITLE
enhanced gallery: fan art

### ISSUE THREAD

**Issue by Vandivier** [Vandivier/arias-tale-rpg#5]
depends on #4 and fan art worth shout out being found there (ideally 3 item minimum
depends on #3 

so this idea is: once there is some good fan art we can give them like official shout outs and spotlights on the website

also, the existing categories should have a filter between canon and fan art once we implement this:
1. lorebook
2. videos
3. pictures

**Comment by Vandivier** [comment]
this is going to make too many files for repo storage

we could do db storage, but users will need to shoulder cost, so it depends on #10 

or we can keep it to the canon proposal process which is already done and potentially extended under #4 (reddit voting about what thing is coolest)

**Comment by Vandivier** [comment]
we should just use reddit for this or smth external if there's interest

top posts we can manually propose inclusion into the repo as data but it wont just be fan art at that point
